extension/src/main/webapp/WEB-INF/conf/gamification/global/completion-configuration.xml

```diff
@@ -169,5 +186,81 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       <type>org.exoplatform.addons.gamification.plugin.GamificationFeaturePlugin</type>
     </component-plugin>
   </external-component-plugins>
+    <external-component-plugins>
```

Comment:
formatting: 2 spaces for a tab instead of 4 please (change IDE settings to use : https://developer.exoplatform.org/#id-ide-settings-to-work-on-exo-platform-projects)